config/routes.rb

```diff
@@ -1,6 +1,12 @@
+require 'sidekiq/web'
+
 Rails.application.routes.draw do
+  authenticate :user, lambda { |u| u.admin? } do
+    mount Sidekiq::Web => '/sidekiq'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.